include/libtorrent/alert_types.hpp

```diff
@@ -1294,12 +1317,17 @@ namespace libtorrent
 
 		// the address libtorrent attempted to listen on
 		// see alert's documentation for validity of this value
-		address address;
+		libtorrent::address address;
```

Comment:
oh, I see, you have a member called "address", that's why you qualify it. another way to do it (I believe) would be to say "class address address;", not sure if that's better though